pkg/client/client.go

```diff
@@ -689,8 +689,8 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		} else if wg := sr.Encryption.Wireguard; wg != nil {
 			ifaces := make([]string, 0, len(wg.Interfaces))
 			for _, i := range wg.Interfaces {
-				iface := fmt.Sprintf("%s (Pubkey: %s, Port: %d, Peers: %d)",
-					i.Name, i.PublicKey, i.ListenPort, i.PeerCount)
+				iface := fmt.Sprintf("NodeEncryption: %s, %s (Pubkey: %s, Port: %d, Peers: %d)",
```

Comment:
This adds the field per device, which I don't think is useful. Please add the NodeEncryption status to the top-level `models.WireguardStatus` object. It's a global property and so it doesn't make sense to repeat it for each interface.

Reply:
I fixed it. Thanks